packages/react/src/SelectPanel/SelectPanel.tsx

```diff
@@ -581,34 +587,41 @@ export function SelectPanel({
           />
           {footer ? (
             <div className={classes.Footer}>{footer}</div>
-          ) : showCancelSaveButtons ? (
+          ) : secondaryAction !== undefined || showCancelSaveButtons ? (
```

Comment:
can we put this in a `hasFooter` var or something just so it's easier to read ?

Reply:
Yeah this was a draft; final implementation is up now!